ribs/visualize.py

```diff
@@ -497,12 +518,13 @@ def parallel_axes_plot(archive,
         archive (ArchiveBase): Any form of archive.
         ax (matplotlib.axes.Axes): Axes on which to plot the parallel 
             coordinates plot. If None,the current axis will be used.
-        bc_order (list of int or list of tuples): By default, the BCs of the 
-            parallel axes will appear in-order. To switch the axes around, 
-            supply a list of the
-            behavior axes (e.g.,``['behavior_1', 'behavior_0', 'behavior_2']``). 
-            It is also possible to plot less behaviors than the number of BCs in
-            the archive (the example also works for archives of 4 BCs).
+        bc_order (list of int or list of tuples): if a list of ints, this
+            specifies the order in which the axes are present (i.e., ``[2, 0,
+             1]``). If it is a list of tuples of the form (int, str), then
+            the order will specify the behavior with the int and the axis label
+            will be the str (e.g., ``[(1, "y-value"), (2, "z-value"),
+             (0, "x-value)]``). The order specified does not need to have the
+            same number of elements as the number of behaviors in the archive.
```

Comment:
Is it more general, in that you can specify as many of a value as you want, like [2,2,2]?

Reply:
We may also want to make it obvious that BCs not included are just left out.